hlx_statics/components/heading.js

```diff
@@ -0,0 +1,48 @@
+import createDivider from './divider.js';
+import decorateLink from './link.js';
+import { createAnchorLink } from '../scripts/lib-adobeio.js';
+
+const headingSizes = ['XL', 'M', 'S', 'XS', 'XXS', 'XXS'];
+
+// Use the Anchor to allow scrolling to heading position minus GlobalNav height
+function Anchor({ id }) {
+  const div = document.createElement('div');
+  div.setAttribute('aria-hidden', 'true');
+  div.setAttribute('id', id);
+  div.style.setProperty('position', 'relative');
+  div.style.setProperty('top', 'calc(-1 * var(--spectrum-global-dimension-size-800)');
+  return div;
+}
+
+export default function decorateHeading(level, block) {
```

Comment:
EDS equivalent of Gatsby [Heading](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L34) component